### ISSUE TITLE
BCUninstaller.dll: System.EntryPointNotFoundException error 

### ISSUE THREAD

**Issue by Komprogue** [Klocman/Bulk-Crap-Uninstaller#495]
Last two (or maybe three) releases of the app always have this error message after every deinstallation process. 
It's running on Windows 7 x64. 

![BCUninstaller_DKOUPBgTgx](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/84424346/30fa94d0-b867-425b-84be-fd049c1461d4)
![BCUninstaller_m9WqT9sD2o](https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/84424346/5020e86d-18bb-458f-ba7c-d05858ea2278)

**Comment by Klocman** [comment]
Run Windows Update and install all system updates.

**Comment by Komprogue** [comment]
> Run Windows Update and install all system updates.

All of them were installed long ago. More likely it's related to something [like this](https://github.com/kobilutil/api-ms-win-core-path-HACK) ([2](https://github.com/nalexandru/api-ms-win-core-path-HACK)), I guess (?)

**Comment by Klocman** [comment]
Yes, it's similar, but this API is added to Windows 7 by Windows Update.
